gmcs/linglib/lexical_items.py

```diff
@@ -970,7 +1087,6 @@ def customize_cops(mylang, ch, lexicon, hierarchies, trigger):
                                      COMPS < [ LOCAL.CAT cat-sat & [ HEAD.PRD +,
                                                            VAL [ SUBJ < >,
                                                                  COMPS < > ] ] ] >,
-                                     SPR < >,
```

Comment:
No verbs should be SPR < > (they all were! spuriously) because adpositions should be able to select for nonemtpy SPR things (otherwise they attach below and above NPs).